### ISSUE TITLE
Add missing focus indicators to test card

### ISSUE THREAD

**Issue by nathancrtr** [CDCgov/prime-simplereport#4021]
## Area or Component
Conduct Test

## Quick Summary
* Visual focus indicator is missing on the checkbox (or perhaps on the underlying input?)
* Visual focus indicator is missing for the result radio buttons

## Accessibility Violation
* `focus-indicator-missing`
    * The visual focus indicator is missing altogether.
    * https://cdc-devtools.dequecloud.com/rule-help/en/focus-indicator-missing

## Action Requested
Add a visual focus indicator to the elements described above.

Things to consider:

![Screen Shot 2022-07-21 at 1.03.41 PM.png](https://images.zenhubusercontent.com/604fb43ab2c0ce6f2a47191e/67d4cc7a-0be8-4d6e-a56c-aa8c3adb0380)

Please consult @vz3 or @kenieh before selecting and implementing a solution.

## Source Code (optional)
https://github.com/CDCgov/prime-simplereport/blob/f11273c49f8e25f8a85449b510e6947a49d3457c/frontend/src/app/testQueue/QueueItem.tsx#L871-L878

https://github.com/CDCgov/prime-simplereport/blob/d4ef63c885670583b20152cada265648c82d1416/frontend/src/app/testResults/CovidResultInputForm.tsx#L65-L89

https://github.com/CDCgov/prime-simplereport/blob/d4ef63c885670583b20152cada265648c82d1416/frontend/src/app/testResults/MultiplexResultInputForm.tsx#L172-L192

**Comment by zdeveloper** [comment]
Not sure why this is flagged, it looks like the focus is set on these elements and it works just fine

you can see the following code that adds the focus behavior in the uswds library `_checkbox-and-radio.scss`
```
.usa-checkbox__input,
.usa-radio__input {
  // The actual input element is only visible to screen readers, because
  // all visual styling is done via the label.
  @include sr-only();

  &:focus {
    @include format-input {
      @include focus-outline(null, null, null, 0.5);
    }
  }
```

![Screenshot from 2022-08-09 10-01-48.png](https://images.zenhubusercontent.com/61095a05dc67526a70b6fd0c/81d94282-a992-41ea-ba1f-608a92ff6488)

![Screenshot from 2022-08-09 10-02-03.png](https://images.zenhubusercontent.com/61095a05dc67526a70b6fd0c/4356a389-7d38-42d6-ac73-c303a12fbd39)

![Screenshot from 2022-08-09 10-02-16.png](https://images.zenhubusercontent.com/61095a05dc67526a70b6fd0c/8d68276a-8824-44fe-be69-226239881cc3)
